src/endpoints/cart.js

```diff
@@ -67,12 +67,14 @@ class CartEndpoint extends BaseExtend {
     )
   }
 
-  BulkAdd(body, options) {
-    const payload = options ? { data: body, options } : body
+  BulkAdd(body) {
     return this.request.send(
       `${this.endpoint}/${this.cartId}/items`,
```

Comment:
@yasiloghmani the remaining linter errors are not resolved here and failing the build. Can you please take a look:
![image](https://user-images.githubusercontent.com/39800563/103552724-57be5480-4e7a-11eb-8649-c60de8351915.png)